diff --git a/HR_timecard_H.py b/HR_timecard_H.py
index acf8022..b0290f4 100644
--- a/HR_timecard_H.py
+++ b/HR_timecard_H.py
@@ -439,10 +439,11 @@ def check_time():
 
     OT = driver.find_element_by_xpath(data["TIMECARD"]["OT"])
     Logging(">> OT Time: " + OT.text)
+    OT_time = OT.text
 
-    return output_clockin,output_clockout,break_time,working_time,clock_in_time,work_method_up,today_work_date
+    return output_clockin,output_clockout,break_time,working_time,clock_in_time,work_method_up,today_work_date, OT_time
 
-def timesheet_list(output_clockin,output_clockout,break_time,working_time):
+def timesheet_list(output_clockin,output_clockout,break_time,working_time, OT_time):
     date_clockin = driver.find_element_by_xpath("//*[@class='admin_status']//div[contains(@class,'daily-wrapper')]/div/div/div/div[2]/div/div[2]/div/div/span")
     #Logging(date_clockin.text)
     date_clock_in = date_clockin.text
@@ -465,9 +466,23 @@ def timesheet_list(output_clockin,output_clockout,break_time,working_time):
             Logging("Clock-out Time is correct")
         else:
             Logging("Clock-out Time is wrong")
+
+        OTtime_li = driver.find_element_by_xpath("//*[contains(@class,'list-table-wrapper')]//div[contains(@col-id,'date') and contains(.,'" + date_clock_in + "')]/following-sibling::div[contains(@col-id,'over_time_label')]/div/div")
+        Logging(OT_time)
+        Logging(OTtime_li)
+        if OT_time == OTtime_li.text:
+            Logging("OT Time is correct")
+        elif OTtime_li.text == "-":
+            Logging("No OT Time")
+        else:
+            Logging("OT Time is wrong")
+            
+                
     else:
         Logging("- Cannot find date")
 
+        
+
     #approval_OT()
 
 def add_event2():
@@ -7823,18 +7838,18 @@ def timecard():
     nightwork()
     breaktime()
     clock_out()
-    output_clockin,output_clockout,break_time,working_time,clock_in_time,work_method_up,today_work_date = check_time()
-    #timesheet_list(output_clockin,output_clockout,break_time,working_time)
-    add_event2()
-    view_details()
-    working_status()
-
-    # day_list = find_date(today_work_date)
-    #work_schedule(day_list)
-    manager()
-    total_manager()
-    # work_correction()
-    delete_punch()
+    output_clockin,output_clockout,break_time,working_time,clock_in_time,work_method_up,today_work_date, OT_time = check_time()
+    timesheet_list(output_clockin,output_clockout,break_time,working_time, OT_time)
+    # add_event2()
+    # view_details()
+    # working_status()
+
+    # # day_list = find_date(today_work_date)
+    # #work_schedule(day_list)
+    # manager()
+    # total_manager()
+    # # work_correction()
+    # delete_punch()
 
 def time_card():
     # # # # Napproval_OT()
@@ -7892,5 +7907,3 @@ def time_card():
     # # # OT_post_midnight()
 
     # work_shift()
-
-#test123
\ No newline at end of file
diff --git a/H_run_file.py b/H_run_file.py
index e363cd3..f2f8ca6 100644
--- a/H_run_file.py
+++ b/H_run_file.py
@@ -54,7 +54,7 @@ def MyExecution(domain_name):
 def My_Execution(domain_name):
     H_functions.access_hr(domain_name)
     HR_timecard_H.timecard()
-    HR_timecard_H.time_card()
+    #HR_timecard_H.time_card()
     #HR_timecard_OT.timecard_OT()
     #HR_timecard_OT.timecard_report()
 
@@ -62,4 +62,3 @@ def My_Execution(domain_name):
 #My_Execution("http:/gw.hanbirolinux.tk/nhr/login")
 My_Execution("http:/global3.hanbiro.com/nhr/login")
 
-# this is a test 123
\ No newline at end of file
